CHANGES.txt

```diff
@@ -1,3 +1,10 @@
+10.15.6 (May 21, 2021)
+ - Updated error handling following the recommended practice of throwing only Error objects.
+ - Updated ioredis version now that their incompatibility with Node <10 was already fixed.
+ - Updated Karma and a transitive dependency with vulnerabilities.
+ - Updated dev dependency on redis-dump to redis-dumpz package to be able to fix a vulnerability on Redis transitive dependency.
+ - Bugfixing - Name two anonymous functions that generate warnigns when transpiling for IE11 (issue #535)
```

Comment:
typo `warnigns` => `warnigns`

Reply:
thanks Emi. late to update in this merge, but will add in the next one for pluggable storage polishing.